Rider Demo/Extensions/UIScrollView+contentSize.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  AddTrackableViewController.swift
+//  Rider Demo
+//
+//  Copyright 2022 Ably Real-time Ltd (ably.com)
+//
+
+import UIKit
+
+extension UIScrollView {
+    func updateContentSize() {
```

Comment:
I think it could be done in some nicer way - just using Interface Builder. Please check `scroll-view` branch in https://github.com/ably/asset-tracking-ios-customer-app-demo

Reply:
A very good catch 👍 Improved it in the latest commit.